main-page.css

```diff
@@ -0,0 +1,134 @@
+.main-page {
+  display: grid;
+  grid-template-rows: 20% 1fr;
+  gap: 5px;
+  position: relative;
+}
+.post-block {
+  display: grid;
+  grid-template-columns: auto 2fr 1fr;
+  justify-content: center;
+  padding-left: 5px;
+  height: auto;
+}
+
+.home {
+  margin-bottom: auto;
+  margin-top: 0ch;
+  top: 0;
+}
+.text-block {
+  position: relative;
+  margin-top: 10ch;
+  margin-right: ;
+  padding-right: 10ch;
+
+  float: inline-end;
+}
+
+.avatar {
+  background-color: white;
+  width: 10ch;
+  height: fit-content;
+  border-radius: 50%;
+  border: solid black 2px;
+  margin-left: auto;
+  width: 8ch;
+}
+.prompt {
+  position: absolute;
+  text-decoration: underline;
+}
+.prompt-line prompt {
+  height: auto;
+  width: 200px;
+  position: absolute;
+}
+.prompt-line a {
+  height: auto;
+  width: auto;
+
+  margin-top: 1ch;
+}
+.post-bar-tweeter {
+  margin-right: auto;
+  align-items: end;
+}
+.post-list-icons {
+  position: absolute;
+  margin-top: 8ch;
+  padding-left: 0;
+  width: 5ch;
+}
+
+.post-bar-icons a {
+  display: inline-block;
+  align-items: center;
+  width: 3ch;
+  margin-top: 7ch;
+}
+.post-bar-icons a:hover {
+  cursor: grab;
+  background-color: blue;
+  border-radius: 50%;
+}
+.tweeter {
+  text-align: center;
+  width: auto;
+  height: fit-content;
+  background-color: rgb(0, 0, 0);
+  margin-top: 8ch;
+  margin-right: 20%;
+  padding-right: 20px;
+  border-radius: 30%;
```

Comment:
A pixel value for border radius might look better without the sides of the oval being chopped off.